ext/node/resolution.rs

```diff
@@ -351,11 +374,15 @@ impl NodeResolver {
         permissions,
       )
       .with_context(|| {
-        format!(
-          "Failed resolving package subpath '{}' for '{}'",
-          package_subpath,
-          package_json.path.display()
-        )
+        if !mode.is_types() {
+          format!(
+            "Failed resolving package subpath '{}' for '{}'",
+            package_subpath,
+            package_json.path.display()
+          )
+        } else {
+          "".to_string()
```

Comment:
What kind of output does this cause? Is this ignored? Maybe we should do something more explicit like:

```rs
let maybe_resolved_path_result = ...;
let maybe_resolved_path = if mode.is_types() {
  maybe_resolved_path_result?
} else {
  maybe_resolved_path_result.with_context(|| {
    // etc...
  })?
};

Reply:
Ah that was what I was looking for. I didn't find a good way to apply `with_context` conditionally.